examples/api/test/widgets/page_view/page_view.0_test.dart

```diff
@@ -49,7 +49,7 @@ void main() {
 
     // Verify that page view index is also updated with same index to page indicator.
     final PageView pageView = tester.widget<PageView>(find.byType(PageView));
-    expect(pageView.controller.page, 1);
+    expect(pageView.controller!.page, 1);
```

Comment:
TODO: Document migration path if in general the change is ok.

Reply:
It is strange google testing passed. 
TODO: test in Google separately.

Reply:
If nothing in Google and no customer_test is failing this wouldn't need a migration.

Reply:
Google: http://cl/597367314